pbft/pbft.cpp

```diff
@@ -303,6 +303,25 @@ pbft::setup_request_operation(const bzn_envelope& request_env, const bzn::hash_t
 void
 pbft::handle_request(const bzn_envelope& request_env, const std::shared_ptr<session_base>& session)
 {
+    if (request_env.timestamp() < (this->now() - MAX_REQUEST_AGE_MS) || request_env.timestamp() > (this->now() + MAX_REQUEST_AGE_MS))
+    {
+        database_msg req;
+        if (session && req.ParseFromString(request_env.database_msg()))
+        {
+            database_response resp;
+            resp.set_allocated_header(new database_header(req.header()));
```

Comment:
you don't need to do this.
resp.mutable_header()->xxxx will allocate automatically. We no longer use set_allocated in the daemon.

Reply:
How do I set the entire header from another header?

Reply:
done

Reply:
*response.mutable_header() = request.header();